### ISSUE TITLE
URL with underscore does not parse

### ISSUE THREAD

**Issue by getreu** [izyuumi/html2md-rs#36]
The following should parse without error:

```html
<!DOCTYPE html><meta http-equiv="content-type" content="text/html; charset=utf-8"><div class="column"><div class="gallery-wrap single">
    <div class="gallery-container">
        <figure class="image">
            <figure class="image">
            <img title="Illustration »Der dunkle Kongress« © ARD / Jürgen Frey"
                 alt="Illustration »Der dunkle Kongress« © ARD / Jürgen Frey" 
                 src="https://hoerspiele.dra.de/fileadmin/www.hoerspiele.dra.de/images/vollinfo/4970918_B01.jpg">
                <figcaption class="image-caption">Illustration »Der dunkle Kongress«
© ARD / Jürgen Frey</figcaption>
</figure></div></div></div>
```

But the parser fails with this error message: 

Malformed attribute: https://hoerspiele.dra.de/fileadmin/www.hoerspiele.dra.de/images/vollinfo/4970918_ - Missing quotation mark at around index 250

Is the underscore in the URL the problem?

**Comment by izyuumi** [comment]
That is a great question 👀 Let me look into this!

**Comment by izyuumi** [comment]
I've figured out the issue: the parser only accepted ASCII input and produced off-by-n errors with non-ASCII input. I will work on a fix.

**Comment by izyuumi** [comment]
Fixed in [v0.10.2](https://github.com/izyuumi/html2md-rs/releases/tag/v0.10.2)
